litmus/PTX/Proxy-MP-gen-sur-surfaceFence1.litmus

```diff
@@ -1,6 +1,6 @@
 PTX Proxy-MP-gen-sur-surfaceFence1
 "MP among generic and surface operations, MP is guaranteed because surface fence is along the base causality path from weak st to suld."
-"surface proxy could be in either of the threads"
+"The surface-proxy could be in either of the threads"
```

Comment:
"surface-proxy" -> "surface barrier" ... it also applies to tests below

Reply:
Updated